# handleException
It's used to handle exception.
